README.md

```diff
@@ -27,70 +31,103 @@ Make sure you get an API key and register a custom domain
 In `configure.swift`:
 
 ```swift
-let mailgun = Mailgun(apiKey: "<api key>", domain: "mg.example.com", region: .eu)
-services.register(mailgun, as: Mailgun.self)
+import Mailgun
+
+// Called before your application initializes.
+func configure(_ app: Application) throws {
+    /// case 1
+    /// put into your environment variables the following keys:
+    /// MAILGUN_API_KEY=...
+    app.mailgunConfiguration = .environment
```

Comment:
Using dot-syntax here would be nice and would provide a place for adding new functionality in the future:
```swift
app.mailgun.configuration = .environment
```

Reply:
It would also be nice if `app.mailgun` were capable of sending emails during app boot / config:

```swift
app.mailgun.send(...)
```

Reply:
Done ✅